patches/net/minecraft/client/multiplayer/MultiPlayerGameMode.java.patch

```diff
@@ -1,16 +1,26 @@
 --- a/net/minecraft/client/multiplayer/MultiPlayerGameMode.java
 +++ b/net/minecraft/client/multiplayer/MultiPlayerGameMode.java
-@@ -120,9 +_,8 @@
+@@ -120,14 +_,15 @@
                  } else if (blockstate.isAir()) {
                      return false;
                  } else {
 -                    block.playerWillDestroy(level, p_105268_, blockstate, this.minecraft.player);
                      FluidState fluidstate = level.getFluidState(p_105268_);
 -                    boolean flag = level.setBlock(p_105268_, fluidstate.createLegacyBlock(), 11);
-+                    boolean flag = blockstate.onDestroyedByPlayer(level, p_105268_, minecraft.player, false, fluidstate);
-                     if (flag) {
-                         block.destroy(level, p_105268_, blockstate);
+-                    if (flag) {
+-                        block.destroy(level, p_105268_, blockstate);
++                    BlockState removedBlockState = block.playerWillDestroy(level, p_105268_, blockstate, minecraft.player);
```

Comment:
Is the reason we are moving this below the `Level#getFluidState` call in case the fluid state changes? If not I would move it back where vanilla has it and just add the capturing of the value in a field there, so that what the patch is changing is more straightforward at a glance when looking at the patch file